opennms-base-assembly/src/main/filtered/etc/thresholds.xml

```diff
@@ -61,6 +61,8 @@
       <expression description="Trigger an alarm when the used JVM heap exceeds 90% of the maximum JVM heap for two samples" type="high" ds-type="node" value="95.0" rearm="85.0" trigger="2" filterOperator="OR" expression="HeapUsageUsed / HeapUsageMax * 100"/>
       <expression description="Trigger an alarm when the ActiveMQ Total Enqueued Messages and Total Dequeued Messages of a queue vary by greater than 15% for two samples" type="high" ds-type="queueMetrics" value="15.0" rearm="10.0" trigger="2" filterOperator="OR" expression="((EnqueCnt + DequeCnt) == 0) ? 0 : (((EnqueCnt - DequeCnt) / ((EnqueCnt + DequeCnt) / 2)) * 100)"/>
       <expression description="Trigger an alarm when the ActiveMQ Total Enqueued Messages and Total Dequeued Messages of a queue vary by greater than 15% for two samples" type="low" ds-type="queueMetrics" value="-15.0" rearm="-10.0" trigger="2" filterOperator="OR" expression="((EnqueCnt + DequeCnt) == 0) ? 0 : (((EnqueCnt - DequeCnt) / ((EnqueCnt + DequeCnt) / 2)) * 100)"/>
+      <expression description="Trigger an alarm when the number of active pollerd threads exceeds 95% of the maximum thread pool size for three samples" type="high" ds-type="node" value="95" rearm="90" trigger="3" filterOperator="OR" expression="ONMSPollerThreadAct / ONMSPollerPoolThrd * 100"/>
```

Comment:
Does this come out to the correct percentage, or does it need to be `(ONMSPollerThreadAct / ONMSPollerPoolThrd) * 100`?

Reply:
```suggestion
      <expression description="Trigger an alarm when the number of active pollerd threads exceeds 95% of the maximum thread pool size for three samples" type="high" ds-type="node" value="95" rearm="90" trigger="3" filterOperator="OR" expression="(ONMSPollerThreadAct / ONMSPollerPoolThrd) * 100"/>
```

Reply:
I think that improves readability anyway, good call.